src/vs/workbench/contrib/chat/browser/media/chatSessions.css

```diff
@@ -133,47 +134,12 @@
 	padding-left: 4px;
 }
 
-.chat-sessions-tree-container .chat-session-item .actions {
```

Comment:
All of these not needed since the defautl behavior of the actions UI/UX is this already